helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -293,31 +293,33 @@ receivers:
         id: parser-containerd
         regex: '^(?P<time>[^ ^Z]+Z) (?P<stream>stdout|stderr) (?P<logtag>[^ ]*) (?P<log>.*)$'
         timestamp:
-          parse_from: time
+          parse_from: attributes.time
           layout: '%Y-%m-%dT%H:%M:%S.%LZ'
       - type: recombine
         id: containerd-recombine
-        combine_field: log
-        is_last_entry: "($$.logtag) == 'F'"
+        combine_field: body.log
+        is_last_entry: "(body.logtag) == 'F'"
       - type: add
         id: containerd-handle_empty_log
         output: filename
-        if: $$.log == nil
-        field: $$body.log
+        if: body.log == nil
+        field: body.log
         value: ""
       {{- end }}
       {{- if or (not .Values.logsCollection.containers.containerRuntime) (eq .Values.logsCollection.containers.containerRuntime "docker") }}
       # Parse Docker format
       - type: json_parser
         id: parser-docker
+        parse_from: body
+        parse_to: body
```

Comment:
Are lines 313 and 314 needed?

Reply:
According to the [doc](https://github.com/open-telemetry/opentelemetry-log-collection/blob/main/docs/operators/json_parser.md), `body` is the default value for `parse_to` and `parse_from`. When I was using default values, I didn't receive the proper event in Splunk (refer to screenshot).
![image](https://user-images.githubusercontent.com/89519921/164606792-32ace773-7b51-4758-ad18-4d5ecb905a75.png)

It was fixed when I provided `parse_to`. Although `parse_from` is not needed.
![image](https://user-images.githubusercontent.com/89519921/164607551-7f17a698-995c-4d74-8124-c5e34d4e77df.png)